src/main/resources/features.xml

```diff
@@ -5,7 +5,7 @@
 
     <feature version="${camel.version.range}">camel</feature>
 
-    <bundle dependency="true">mvn:org.apache.commons/commons-codec/${commons.codec.version}</bundle>
+    <bundle dependency="true">mvn:commons-codec/commons-codec/${commons.codec.version}</bundle>
```

Comment:
I don't see `commons-codec` in the `pom.xml` file. Is it even used in fcrepo-camel?